communication/state_telemetry.cpp

```diff
@@ -251,6 +262,48 @@ static mav_result_t state_telemetry_set_arm_from_cmd(State* state, mavlink_comma
     return result;
 }
 
+static mav_result_t state_telemetry_send_autopilot_capabilities(State* state, mavlink_command_long_t* packet)
+{
+    mav_result_t result = MAV_RESULT_ACCEPTED;
+
+    // Autopilot version
+    if (packet->param1 == 1)
+    {
+        const uint8_t flight_custom_version[8]      = "MAVRIC";
+        const uint8_t middleware_custom_version[8]  = "master";
```

Comment:
Yes, that would be possible via a #define set by the Makefile, and with a default value in the code. I opened issue #219